core/app/models/mno_enterprise/app.rb

```diff
@@ -1,69 +1,17 @@
-# == Schema Information
-#
-# Endpoint: /v1/apps
-#
-#  id                       :integer         not null, primary key
-#  nid                      :string         e.g.: 'wordpress'
-#  name                     :string(255)
-#  description              :text
-#  created_at               :datetime        not null
-#  updated_at               :datetime        not null
-#  logo                     :string(255)
-#  version                  :string(255)
-#  website                  :string(255)
-#  slug                     :string(255)
-#  categories               :text
-#  key_benefits             :text
-#  key_features             :text
-#  testimonials             :text
-#  worldwide_usage          :integer
-#  tiny_description         :text
-#  popup_description        :text
-#  stack                    :string(255)
-#  terms_url                :string(255)
-#  tags                     :text
-#  rank                     :integer
-#
-
 module MnoEnterprise
   class App < BaseResource
-    scope :active, -> { where(active: true) }
```

Comment:
I agree. One of the missing feature of https://github.com/chingor13/json_api_client